models/terminator/model.sdf

```diff
@@ -45,8 +45,8 @@
           </script>
         </material>
         <plugin name="lod" filename="libHeightmapLODPlugin.so">
-          <lod>3</lod>
-          <skirt_length>0.1</skirt_length>
+          <lod>0</lod>
```

Comment:
What is this change doing for the PR exactly?

Reply:
Sorry, this was only slightly related to this PR. These changes make this terrain model consistent with our other terrain models. lod = 0 will make gazebo show maximum terrain geometric resolution. skirt_length = 0 will remove terrain skirts, which can cause unwanted shadows and are not needed with lod = 0.